### DISCUSSION TITLE
Fscrawler version 2.7 install on RHEL8 log files fscrawler.log and confg folder missing

### DISCUSSION THREAD

**Discussion by rcheatham1** [dadoonet/fscrawler#1234] [Q&A]
Hello,

We currently have Fscrawler version 2.7 installed on RHEL8 server.  According to the documentation a fscawler.log file is created when the software gets installed.  I'm  new to the Linux environment but I've search everywhere and the log file doesn't exist. I look in the directory when its installed and I only have a bin and lib directory.  Can someone help.

**Comment by dadoonet** [comment]
Not when it gets installed but when it runs.

**Reply by rcheatham1** [reply]
Thanks but we can't locate the fscrawler.log after were run the jobs.  We looked in the directory where fscrawer was installed and we only see the bin and lib directory.  The only thing we can see when a job is run the the _status.json file being created/ updated which is located in the job directory.

**Reply by dadoonet** [reply]
What exact command line are you running?

**Comment by rcheatham1** [comment]
This is the command we run  

sudo /opt/fscrawler/bin/fscrawler --config_dir /etc/fscrawler qp --debug

**Reply by dadoonet** [reply]
What is the job setting file?

**Reply by rcheatham1** [reply]
Sorry for the late response.  We decided to download the latest version of fscrawler , and once we installed it this morning  and now we have the config folder and logs files.  We are running into another issue when we use the pipeline setting in our job. I will close this issue and open a new one.    Thanks
